Automattic/jetpack
# Calendly: Add CSS to adjust the Calendly block to the masterbar

Fixes #14365

#### Changes proposed in this Pull Request:

* This adds CSS to the frontend of the site to adjust the close button in the Calendly widget to the height of the master bar

#### Testing instructions:
* Add a Calendly block to a post and set the style to "link"
* Open the post in the front end
* Click on the button to open the widget
* Check you can see the close button:
<img width="1440" alt="Screenshot 2020-01-17 at 11 30 44" src="https://user-images.githubusercontent.com/275961/72609448-d5611b80-391c-11ea-9b58-83c185b287c6.png">

#### Proposed changelog entry for your changes:

* no changelog entry needed
